source/MRMesh/MRLineObject.h

```diff
@@ -40,6 +41,10 @@ class MRMESH_CLASS LineObject : public ObjectLinesHolder
     MRMESH_API void setCenter( const Vector3f& center );
     /// updates xf to scale size
     MRMESH_API void setSize( float size );
+    /// calculates line size from xf
+    MRMESH_API float getSize( void ) const;
```

Comment:
we typically write `()` instead of `( void )`

Reply:
ok